src/components/cylc/tree/TreeItem.vue

```diff
@@ -133,8 +129,7 @@ export default {
   ],
   components: {
     task: Task,
-    job: Job,
-    CylcObject: CylcObject
```

Comment:
Half of the `CylcObject` is going to the `cylc-object` directive. More specifically, the half part that listens for `click` on the Cylc object component and triggers the menu to display.

The other part, the actual menu and code to display it, will go to a different component, loaded only once in each view.